libs/shared/contentful/src/lib/queries/purchase-with-details-offering-content/factories.ts

```diff
@@ -0,0 +1,87 @@
+import { faker } from '@faker-js/faker';
```

Comment:
Nit: non-blocker - should this file be a part of `libs/shared/contentful/src/lib/factories.ts` or should we break apart that file and have factories within their corresponding query? I prefer the latter.

Reply:
Aah, I see Julian made a comment on another PR with the same preference. Please disregard this nit. :)